app/src/main/java/nl/tudelft/bbw/blockchain/Block.java

```diff
@@ -280,19 +280,36 @@ public boolean verifyBlock(Object o) {
     @Override
     public String toString() {
         return "Block#" + getSequenceNumber() + "{owner=" + owner.getName() + ",contact="
-                + contact.getName() + "}";
+                + contact.getName() + (isRevoked() ? ",REVOKE" : "") + "}";
     }
 
     /**
+     * ToString method designed for Interaction Design Module
+     */
+    public String toString2() {
```

Comment:
this method is used for the interaction design, since it is not needed this method is removed too.